src/pcp/GNUmakefile

```diff
@@ -25,6 +25,7 @@ SUBDIRS = \
 	iostat \
         ipcs \
 	mpstat \
+	meminfo \
```

Comment:
This list is in alphabetic order IIRC - if so, this line needs to be up one spot.